### PR TITLE
GTK4: fix link styles

### PR DESCRIPTION
**Pull request by danrabbit**
* Make sure we don't style the entire label with accent color when it contains a link
* Add underline to links

### CODE REVIEW

File: src/gtk-4.0/_typography.scss [hunk 1]
```diff
@@ -40,8 +40,8 @@ label.h4,
 }
 
 .accent,
```
Comment (cassidyjames) [inline src/gtk-4.0/_typography.scss:42]:
```suggestion
.accent {
```

File: src/gtk-4.0/_typography.scss [hunk 2]
```diff
@@ -40,8 +40,8 @@ label.h4,
...
-.link,
-:link {
+button.link,
+link {
```
Comment (cassidyjames) [inline src/gtk-4.0/_typography.scss:44]:
```suggestion
```

File: src/gtk-4.0/_typography.scss [hunk 3]
```diff
@@ -134,6 +134,11 @@ label.h4,
     }
 }
 
+button.link,
+link {
+    text-decoration: underline;
```
Comment (cassidyjames) [inline src/gtk-4.0/_typography.scss:139]:
```suggestion
    @extend .accent;
    text-decoration: underline;
```

**Review by cassidyjames — CHANGES_REQUESTED** [review]
So this is a tiny thing, but keeps the Sass more in sync in my experience: you can use `@extend` to ensure one selector stays in sync with the other, which means you don't have to make sure to update/change multiple places in the future.

It's optional, but in reading this change it stood out to me.

**Comment by danrabbit** [comment]
@cassidyjames I've had some issues with extends pulling in styles where I don't want them, but this seems reasonable :+1:

**Review by cassidyjames — APPROVED** [review]

**Pull request merged**